requirements/openvino.txt

```diff
@@ -3,5 +3,5 @@
 nncf==2.3.0
 onnx==1.10.1
 openmodelzoo-modelapi==2022.2.0
```

Comment:
Please try the next version that I've just uploaded :)
![image](https://user-images.githubusercontent.com/4629606/210287612-db842dcb-3d57-4f07-9d53-8106e0cdaff6.png)